com.expidusos.file_manager.yml

```diff
@@ -0,0 +1,33 @@
+app-id: com.expidusos.file_manager
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+separate-locales: false
+command: file_manager
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --socket=pulseaudio
+  - --share=network
+  - --own-name=com.expidusos.file_manager
```

Comment:
you don't need this one, it is automatically granted.

Reply:
Alright.